src/components/shared/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+
+import { Button as OriginalButton } from '@dooboo-ui/native';
```

Comment:
Providing things like this would make users complicated.
Let's try to make it simple and guide users who're willing to use `@dooboo-ui/native` just a simple approach.